src/dynsight/_internal/analysis/rdf.py

```diff
@@ -0,0 +1,179 @@
+from __future__ import annotations
+
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+from ovito.io import import_file
+from ovito.modifiers import (
+    CoordinationAnalysisModifier,
+    DeleteSelectedModifier,
+    ExpressionSelectionModifier,
+    LoadTrajectoryModifier,
+    TimeAveragingModifier,
+)
+from scipy.signal import find_peaks
+
+
+class RDF:
+    """Object for computing and analyzing the Radial Distribution Function.
+
+    The Radial Distribution Function (RDF) describes how particle density
+    varies as a function of the distance.
+
+    Parameters:
+        trajectory (list[Path]):
```

Comment:
First, remove types from doc strings. It should just be:

```python
trajectory:
    comment.
```

Second: this type does not match the typing below, which is just `Path`.